src/lesson6tasks/Point.java

```diff
@@ -0,0 +1,14 @@
+package lesson6tasks;
+
+public class Point {
+    public Point (double x1, double y1, double x2, double y2) {
+        x1 = 2;
+        y1 = 5;
+        x2 = 4;
+        y2 = 6;
+    }
```

Comment:
нижче я привів приклад як повинен клас точка описуватися полями х та у
```suggestion
public class Point {
    private double x;
    private double y;
    public Point (double x, double y) {
        this.x = x;
        this.y = y;
    }
```